Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabSaveLoadHandler.cpp

```diff
@@ -613,10 +624,10 @@ namespace AzToolsFramework
                 contentLayout->addWidget(footerSeparatorLine);
 
                 QLabel* prefabSavePreferenceHint = new QLabel(
-                    "<u>You can prevent this window from showing in the future by updating your global save preferences.</u>",
+                    "You can prevent this from showing in the future by updating your preferences.",
```

Comment:
nit: Might be a ux question but if you remove the underline, do you think it's obvious for the user to hover over the message to see the tool tip. Having the underline makes the user think that the text is either clickable or has more metadata. I remember trying to put the underline on only 'preferences' but couldn't find an easy way to do that. So I ended up underlining the whole message, which is not elegant but does the job I think

Reply:
I discussed it with Josh, and determined that the underline makes it look like it's clickable, so that was the concern.
Maybe we could add an icon to signify that there's a tooltip?

Reply:
Yes, I think anything other than a plain label would do the trick. A question mark or info icon would help I think